src/sbl_filing_api/services/submission_processor.py

```diff
@@ -99,47 +100,56 @@ async def get_from_storage(period_code: str, lei: str, file_identifier: str, ext
         ) from e
 
 
-async def validate_and_update_submission(period_code: str, lei: str, submission: SubmissionDAO, content: bytes):
+async def validate_and_update_submission(
+    period_code: str, lei: str, submission: SubmissionDAO, content: bytes, exec_check: dict
+):
     validator_version = imeta.version("regtech-data-validator")
     submission.validation_ruleset_version = validator_version
     submission.state = SubmissionState.VALIDATION_IN_PROGRESS
     submission = await update_submission(submission)
```

Comment:
why isn't this one in the with context block?

Reply:
Because I was lazy? Updated.  Removed the concept of the update function and the get_submission function from having a None session.  Calling code will now be expected to handle that piece.